### DISCUSSION TITLE
Failed to validate Authorization header with basic scheme

### DISCUSSION THREAD

**Discussion by varazir** [authelia/authelia#7478] [Troubleshooting]
Hello. 
I have a external service that need to access a internal service, it don't support any proxy user authentication. 
Befor  it worked by setting a bypass policy and networks. 

Now I just get error 

`time="2024-07-02T10:31:43+02:00" level=debug msg="Error occurred while attempting to authenticate a request but the matched rule was a bypass rule" error="failed to validate Authorization header with basic scheme: validated parsed credentials of Authorization header but they are not valid for user 'xxxx': %!w(<nil>)" method=HEAD path=/api/authz/forward-auth remote_ip=xxxxxxxxxxxx`

I'm fronting authelia with a HAproxy

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

Hola,

Parece que estás enfrentando un problema con la validación del encabezado de autorización en tu configuración actual de Authelia. El error indica que, aunque la regla de bypass se ha aplicado, la validación del encabezado de autorización con el esquema básico ha fallado.

Aquí hay algunas cosas que podrías verificar y probar:

1. **Revisar la Configuración de la Regla de Bypass**:
   Asegúrate de que la regla de bypass esté correctamente configurada en tu archivo de configuración de Authelia. Asegúrate de que las redes y dominios especificados sean correctos.

2. **Encabezados de Autorización**:
   Verifica que el servicio externo no esté enviando encabezados de autorización que puedan estar interfiriendo con la regla de bypass. Si es posible, configura el servicio para que no envíe ningún encabezado de autorización.

3. **Actualizar la Configuración de Authelia**:
   Asegúrate de que tu configuración de Authelia esté actualizada y que no haya cambios recientes que puedan haber afectado la forma en que se manejan las reglas de bypass.

4. **Logs Detallados**:
   Revisa los logs detallados de Authelia para obtener más información sobre por qué la validación del encabezado de autorización está fallando. Esto puede proporcionar pistas adicionales sobre lo que está mal.

5. **Documentación y Ejemplos**:
   Consulta la documentación oficial de Authelia para ver ejemplos de configuración de reglas de bypass y asegurarte de que tu configuración esté alineada con las mejores prácticas.

Aquí tienes algunos enlaces a la documentación que podrían ser útiles:
- [Authelia y NGINX](https://www.authelia.com/integration/proxies/nginx)
- [Authelia y Traefik](https://www.authelia.com/integration/proxies/traefik)

Si necesitas más ayuda, por favor proporciona más detalles sobre tu configuración actual y cualquier cambio reciente que hayas realizado.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Reply by varazir** [reply]
I'm using pfSense/HAproxy setup to front authelia
Why do you not reply in English ?

**Comment by varazir** [comment]
I'm using pfSense/HAproxy setup to front authelia

**Comment by varazir** [comment]
My conf 
https://gist.github.com/varazir/d0e0a6ca18fc0ec92f8dfe0debfc4320

**Comment by varazir** [comment]
I found another solution.
